fenzo-core/src/main/java/com/netflix/fenzo/TaskScheduler.java

```diff
@@ -466,6 +484,9 @@ private TaskScheduler(Builder builder) {
                 autoScaler.setDelayScaleDownBySecs(builder.delayAutoscaleDownBySecs);
             if(builder.delayAutoscaleUpBySecs > 0L)
                 autoScaler.setDelayScaleUpBySecs(builder.delayAutoscaleUpBySecs);
+            if (builder.disabledVmDurationInSecs > 0L) {
+                autoScaler.setDisabledVmDurationInSecs(builder.disabledVmDurationInSecs);

```

Comment:
added javadoc to outline that should value should be greater than the cooldown period and is only useful if the autoscaling action may take more time to execute than your cooldown period.